### ISSUE TITLE
Async Version?

### ISSUE THREAD

**Issue by samcov** [martinjw/dbschemareader#176]
Is there any chance of ever getting an async version of this library?

**Comment by martinjw** [comment]
Yes, will probably do it sometime. It's a lot of work, but not difficult work.

**Comment by samcov** [comment]
I know, it's a major pain, straightforward, but a lot of work & testing! :)

No rush, just asking.

**Comment by swidz** [comment]
+1
